### PR TITLE
chore: replace deprecated endpoints for vespa feeding

### PR DESCRIPTION
**Pull request by esolitos**
## What

- Replace the custom vespa cloud endpoints with the generated ones.

## Why

- The custom endpoints are deprecated and will be removed in the future.

---
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Review by kkraune — APPROVED** [review]
👍

File: _config.yml [hunk 1]
```diff
@@ -58,19 +58,26 @@ search:
...
         - open_index.json
-    - url: https://vespacloud-docsearch.vespa-team.aws-eu-west-1a.z.vespa-app.cloud/
+    # vespacloud-docsearch // aws // us-west-1a
+    - url: https://ed053d52.b68a8c0d.z.vespa-app.cloud/
       indexes:
         - open_index.json
-    - url: https://enclave.vespacloud-docsearch.vespa-team.aws-us-east-1c.z.vespa-app.cloud/
+        
+    # vespacloud-docsearch // aws (enclave) // us-west-1a
+    - url: https://f10d3607.b68a8c0d.z.vespa-app.cloud/
```
Comment (andreer) [inline _config.yml:75]: This actually points to us-east-1c

Comment (esolitos) [reply _config.yml:75]: Thanks for double checking 👍🏼
Endpoint is right, but comment is wrong. Fixing it in a new PR. 😊

File: _paragraphs_config.yml [hunk 2]
```diff
@@ -6,18 +6,23 @@ search:
...
-    - url: https://vespacloud-docsearch.vespa-team.gcp-us-central1-f.z.vespa-app.cloud/
+        - paragraph_index.json
+    # vespacloud-docsearch // gcp // us-central1-f
+    - url: https://a341952a.b68a8c0d.z.vespa-app.cloud/
       indexes:
-        - paragraph_index.json 
-    - url: https://vespacloud-docsearch.vespa-team.aws-eu-west-1a.z.vespa-app.cloud/
+        - paragraph_index.json
+    # vespacloud-docsearch // aws // us-west-1a
+    - url: https://ed053d52.b68a8c0d.z.vespa-app.cloud/
```
Comment (andreer) [inline _paragraphs_config.yml:18]: points to eu-west-1a (but comment says us)

**Pull request merged**